server/src/analytics/GoogleAnalytics.ts

```diff
@@ -1,26 +1,21 @@
 /* istanbul ignore file: external system */
 import * as os from "os";
 import got from "got";
-import * as qs from "qs";
 import { ServerState } from "../types";
-import {
-  Analytics,
-  AnalyticsPayload,
-  DefaultRawAnalyticsPayload,
-  RawAnalyticsPayload,
-} from "./types";
+import { Analytics, AnalyticsPayload } from "./types";
 
-const GOOGLE_ANALYTICS_URL = "https://www.google-analytics.com/collect";
+const GA_URL = "https://www.google-analytics.com/mp/collect";
+const GA_SECRET = "5ddn8Rl5QwKk51Uls_BM6A";
```

Comment:
We should consider how we inject this. I think we injected the GA app identifier from the top level index file; but maybe we should push further and have it loaded from a json file injected in the packaging process? Though we would probably need a switch for development.

The reason I bring it up, is I suspect we get some false positives on foundry because someone has forked the code but still kept the analytics ids the same.

Reply:
Good point. You are suggesting something like reading a non-versioned secret file or an env variable at the time of packaging ?

Reply:
I am not sure. Maybe having a constants file pulled in at the top index with an empty set of default values. In development read the values from `env` variables, and during packaging write out the `env` variables to the constants file.

So anyone forking the codebase will get the empty version, but we can setup the environment variables for local development and release with no loss - we get analytics in production and in local development.

Maybe. Open to other approaches?

Reply:
We have created a constants file that has empty default values, but optionally populates using env variables.

We therefore don't keep our analytics codes in the repo but can include them during the build.